packages/tools/pluggable-widgets-tools/src/web/document.ts

```diff
@@ -0,0 +1,239 @@
+export function unBlockAbsoluteElement(
```

Comment:
There's a lot of untested code here, although it's not very complicated I consider this brittle. I suggest bringing it up with Diego/Mitya to determine if it's worth the time.